src/html.sortable.js

```diff
@@ -557,7 +560,7 @@ var sortable = function (sortableElements, options) {
     // Handle drag events on draggable items
     _on(items, 'dragstart', function (e) {
       e.stopImmediatePropagation()
-      if ((options.handle && !_matches(e.target, options.handle)) || this.getAttribute('draggable') === 'false') {
+      if ((options.handle && !_matches(e.target, options.handle)) || this.getAttribute('draggable') === 'true') {
```

Comment:
Sorry, I have to bug you one last time. This one was a mistake 😉. You need to keep this as `false` you wanted to change the value below.